Dockerfile

```diff
@@ -9,7 +9,8 @@ FROM docker-all.repo.sonatype.com/node:12
 
 RUN apt-get update && \
     apt-get autoremove -y && apt-get clean -y && \
-    apt-get install jq -y
+    apt-get install jq -y && \
+    apt-get install chromium libatk-bridge2.0-0 libxkbcommon0 libgbm1 -y
```

Comment:
We use puppeteer for running an automated browser.  Although puppeteer installs its own copy of chromium, we have to make sure the necessary system libraries are in place